src/types/organization.ts

```diff
@@ -83,3 +86,46 @@ export function fromPublicAdministrationToFoundAdministration(
     value => value
   );
 }
+
+export function fromOrganizationInstanceToOrganizationObject(
+  organizationInstance: OrganizationModel
+): Either<Errors, Organization> {
+  const legalRepresentative = {
+    email: organizationInstance.legalRepresentative.email,
+    family_name: organizationInstance.legalRepresentative.familyName,
+    fiscal_code: organizationInstance.legalRepresentative.fiscalCode,
+    given_name: organizationInstance.legalRepresentative.givenName,
+    phone_number: organizationInstance.legalRepresentative.phoneNumber,
+    role: organizationInstance.legalRepresentative.role
+  };
+  const users =
+    organizationInstance.users &&
+    organizationInstance.users.map(user => ({
+      email: user.email,
+      family_name: user.familyName,
+      fiscal_code: user.fiscalCode,
+      given_name: user.givenName,
+      role: user.role,
+      work_email: user.workEmail || undefined
```

Comment:
why we need the `|| undefined` part ?

Reply:
The `workEmail` property is optional in the organization object, so its value must be an email string or `undefined`. The queries always return the stored value from the db, which will be the email string or a `null` value, so in this second case we must use `undefined` in order to create a valid organization object.